app/forms/claimants_detail.rb

```diff
@@ -9,4 +9,19 @@ class ClaimantsDetail < BaseForm
   attribute :continued_employment, :boolean
   attribute :agree_with_claimants_description_of_job_or_title, :boolean
   attribute :disagree_claimants_job_or_title, :text
+
+  def to_h # rubocop:disable Metrics/MethodLength
```

Comment:
I think you have to re enable a rubocop cop afterwards ?

Reply:
I created a new method below, added too many lines and rubocop flagged it without needing to be re-enabled. :)

Reply:
oh good - thanks for checking